Apollonius_graph_2/include/CGAL/Parabola_2.h

```diff
@@ -59,7 +59,7 @@ class Parabola_2
 #else
   static const FT& STEP()
   {
-    static FT step_(2);
+    static const FT step_(2);
```

Comment:
We can do that. I guess this is meant as an acceleration for a more complicated FT.
Do you have a preference?